doc/release/1.16.0-notes.rst

```diff
@@ -106,3 +106,8 @@ raise a ``DeprecationWarning`` if the array is not numerical (i.e.,
 if ``np.positive(array)`` raises a ``TypeError``. For ``ndarray``
 subclasses that override the default ``__array_ufunc__`` implementation,
 the ``TypeError`` is passed on.
+
+Arrays based off buffers cannot be set ``writeable``
```

Comment:
Based off readonly buffers? Seems harmless to allow them to be made readonly if the source buffer was writeable

Reply:
fixed